lib/handlers/run.js

```diff
@@ -80,6 +80,17 @@ function run(req, res, next, options) {
             // now, execute the handler block. it's async.
             vasync.forEachPipeline({
                 func: function runHandlers(fn, handlerCb) {
+                    // Ensure that invalid handlers aren't run and throwing
+                    // cryptic errors
+                    if (!_.isFunction(fn)) {
```

Comment:
Nice catch. Any reason we can't assert at some point during conductor creation instead of waiting till run time?